src/components/sidebar/SafeListContextMenu/MultiAccountContextMenu.tsx

```diff
@@ -0,0 +1,107 @@
+import type { MouseEvent } from 'react'
+import { useState, type ReactElement } from 'react'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import IconButton from '@mui/material/IconButton'
+import MoreVertIcon from '@mui/icons-material/MoreVert'
+import MenuItem from '@mui/material/MenuItem'
+import ListItemText from '@mui/material/ListItemText'
+
+import EntryDialog from '@/components/address-book/EntryDialog'
+import EditIcon from '@/public/images/common/edit.svg'
+import PlusIcon from '@/public/images/common/plus.svg'
+import ContextMenu from '@/components/common/ContextMenu'
+import { trackEvent, OVERVIEW_EVENTS, OVERVIEW_LABELS } from '@/services/analytics'
+import { SvgIcon } from '@mui/material'
+import { AppRoutes } from '@/config/routes'
+import router from 'next/router'
+import { CreateSafeOnNewChain } from '@/features/multichain/components/CreateSafeOnNewChain'
+
+enum ModalType {
+  RENAME = 'rename',
+  ADD_CHAIN = 'add_chain',
+}
+
+const defaultOpen = { [ModalType.RENAME]: false, [ModalType.ADD_CHAIN]: false }
+
+const MultiAccountContextMenu = ({
+  name,
+  address,
+  chainIds,
+}: {
+  name: string
+  address: string
+  chainIds: string[]
+}): ReactElement => {
+  const [anchorEl, setAnchorEl] = useState<HTMLElement | undefined>()
```

Comment:
```suggestion
  const [anchorEl, setAnchorEl] = useRef<HTMLElement>(null)
```

Reply:
Hmm internally we use the Mui Menu component (https://mui.com/material-ui/react-menu/)
All examples how to use it use the anchorEl property and do not work with refs. I think this is the intended way. 

I will leave it as is for now. If you feel like it could be improved we could do it separately as we use this ContextMenu component in other places in the app as well :)

Reply:
Hmmm got it 👍